CHANGELOG.md

```diff
@@ -7,7 +7,13 @@
   This attribute represents the pre-GA name for "Uniform Bucket Level Access"
   becoming GA. Applications should use `uniform_bucket_level_access` instead.
 
-## v1.23.0 - TBD
+## v1.24.0 - TBD
+
+## v1.23.0 - 2021-01
+
+### Bigtable
```

Comment:
FYI I'm not sure if this should be under "Bigtable" since that's apparently the only thing affected or "Common Libraries" since that where the changes actually are?  Anyone have an opinion?

Reply:
I'd say "Common Libraries" given that's where the changes are.

Reply:
Done